frontend/app/templates/work_packages.list.new.html

```diff
@@ -0,0 +1,71 @@
+<div cg-busy="vm.loaderPromise" class="work-packages--details-content -create-mode">
+  <div ng-if="vm.workPackage">
+    <h2>{{ ::I18n.t('js.work_packages.create.header') }}</h2>
+
+    <div class="single-attribute subject">
```

Comment:
Well, it seems that this is due to the differences in styling of the different tabs. As the create tab does not have tabs, @0xF013 introduced a modification to deal with that fact.

It's a long term symptom of the _very different_ form views for create and edit.